PullRequestSandbox/Program.cs

```diff
@@ -10,7 +10,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            /// yepyep
+            // this line will have lots of comments
```

Comment:
## IV. THE CORE OF THE SIMULATION ARGUMENT
 
The basic idea of this paper can be expressed roughly as follows: If there were a substantial chance that our civilization will ever get to the posthuman stage and run many ancestor-simulations, then how come you are not living in such a simulation?
            We shall develop this idea into a rigorous argument. Let us introduce the following notation:
 
: Fraction of all human-level technological civilizations that survive to reach a posthuman stage
 
: Average number of ancestor-simulations run by a posthuman civilization
 
*: Average number of individuals that have lived in a civilization before it reaches a posthuman stage
 
The actual fraction of all observers with human-type experiences that live in simulations is then
 
 
 
Writing  for the fraction of posthuman civilizations that are interested in running ancestor-simulations (or that contain at least some individuals who are interested in that and have sufficient resources to run a significant number of such simulations), and  for the average number of ancestor-simulations run by such interested civilizations, we have
 

 
and thus:
 
                                        (*)
 
Because of the immense computing power of posthuman civilizations,  is extremely large, as we saw in the previous section. By inspecting (*) we can then see that at least one of the following three propositions must be true:
 
(1)        
(2)        
(3)